lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -190,6 +191,8 @@ def run!(verify_result: true, allow_protected_ns: false, prune: true)
 
     private
 
+    attr_accessor :kubeconfig
```

Comment:
This is a crappy API, fighting against the fact that `KubeclientBuilder` is a mixin when it should probably be a collaborating object. I'll prototype a fix as a second PR against this branch to see what you think.

Reply:
Fixed in https://github.com/powerhome/kubernetes-deploy/pull/1

Reply:
the KubeclientBuilder-as-object makes sense to me and conceptually makes more sense to me. I'm ok with the change, wdyt @dturn ?